withdraw-by-check/withdraw-by-check-json-contract.md

```diff
@@ -102,13 +102,13 @@ Unless otherwise noted, all values will be passed as double-quote encapsulated c
 | 501        | Avail. Balance <= $0.00 || Target Share/Loan available balance<=$0.00 |
 | 502        | Invalid Address |||
 | 503        | Account Not Found || Error Code 503 is used for account find/validate errors |
-|            | Account Warning Found |||
+|            | Account Warning Found List [3-digit comma separated account warning list] |||
```

Comment:
Use the same pattern that is already used to append extra information to the error message.  See the modifier in other error messages.  Same thing applies to the other changes in this file.

Reply:
made suggested change